docs/app/Components/ComponentDoc/ComponentDescription.js

```diff
@@ -28,7 +28,7 @@ export default class ComponentDescription extends Component {
   renderSemanticDocsLink = () => {
     const { _meta } = this.props
 
-    if (!META.isSemanticUI(_meta)) return null
+    if (META.isAddon(_meta)) return null
```

Comment:
`isSemanticUI` and `isStardust` is now determined by whether or not a component is an `addon` type.  All other types are SUI core.